cli/args/flags.rs

```diff
@@ -391,6 +391,7 @@ pub struct Flags {
   pub ext: Option<String>,
   pub ignore: Vec<PathBuf>,
   pub import_map_path: Option<String>,
+  pub env_file: Option<String>,
```

Comment:
> That was also one of the requirements for this feature - no magic discovery, not walking up the file system.

Generally a `.env` file would be stored right beside a deno.json. If a deno.json exists, maybe we should traverse up and stop at the deno.json directory when someone provides `--env` with no path?